Makefile

```
@@ -0,0 +1,12 @@
+.DEFAULT_GOAL := help
+.PHONY: help
+
+emojis: example/img/emojis ## Install our emojis
+
+example/img/emojis: node_modules/@readme/emojis
```

Does this need to be split in to two tasks?

```suggestion
emojis: ## Install our emojis
	node_modules/@readme/emojis
```

---

Splitting it in two was so:

* we can type `make emojis`
* it only runs if `example/img/emojis` is older than `node_modules/@readme/emojis`

But then again, it probably doesn't matter.

---

lol no yours is better; like I said, I know not about makefiles!

---

Heh, I think the original use case was like incrementally compiling c?

```
program.exe: program.o library.o
  gcc --some command

program.o: program.c
  gcc --some other command

library.o: library.c
  gcc --some other other command
```

If you made some change to `program.c` and ran `make` it would skip recompiling library.c?

---

Ahh, I gotchya. I don't think that kind of smart comparison would apply in our case though, since we manually running that make task as a one-off? But like you said: "it ~probably~_definitely_ doesn't matter".